subprojects/configuration-cache/src/main/kotlin/org/gradle/configurationcache/ConfigurationCacheBuildOperations.kt

```diff
@@ -31,7 +31,7 @@ fun <T : Any> BuildOperationExecutor.withLoadOperation(block: () -> T) =
 
 internal
 fun BuildOperationExecutor.withStoreOperation(cacheKey: String, block: () -> Unit) =
-    withOperation("Store configuration cache state $cacheKey", block, StoreDetails, StoreResult)
+    withOperation("Store configuration cache state", block, StoreDetails, StoreResult)
```

Comment:
@adammurdoch, I guess the preferred implementation would be to have the `cacheKey` as part of `StoreDetails`, right?